### ISSUE TITLE
Add a section of known failures in tutorial on NVIDIA driver

### ISSUE THREAD

**Issue by lebensterben** [clearlinux/clear-linux-documentation#919]
There are certain known cases when NVIDIA driver is not likely to be successfully installed. And there are known bugs in specific kernel version that the NVIDIA driver won't function properly.

By the nature of GPU driver, when it fails, the user is likely to panic. And fixing the system is doable, but it's just annoying.

I suggest to include a section to record all known issues in the documentation. For example, I think we need to list kernel versions that the driver won't work, and also mention that Optimus-enabled laptops are very not likely to work properly.

These known issues could come from the community forum and bug reports in `clearlinux/distribution`. And in the very beginning of the tutorial, I suggest to emphasize the chance of failures, and guide users to review the known issues before they proceed to install the driver.

**Comment by puneetse** [comment]
> There are certain known cases when NVIDIA driver is not likely to be successfully installed.

Which cases were you thinking specifically?

> For example, I think we need to list kernel versions that the driver won't work

I kind of wanted to avoid compiling a dataset of nvidia-system bugs in the documentation for some reasons:

Other parts of the Internet are already try to do this and it is very easy for datapoints to go stale with no followup, which can double down on the confusion when trobuleshooting. 

With that said, the documentation is really there to serve users. If someone were to take the time to contribute and maintain it in the documentation that would be lovely, but I think a more dynamic place like the forums is better to start. 



>  also mention that Optimus-enabled laptops are very not likely to work properly.

The latest version does have a big note about optimus systems specifically.If more is needed, please highlight it.

**Comment by mvincerx** [comment]
@puneetse Please follow up per meeting.

**Comment by lebensterben** [comment]
> And there are known bugs in specific kernel version that the NVIDIA driver won't function properly.

Actually I made a wrong statement here. I was referring to https://github.com/clearlinux/distribution/issues/1133, and that is not really a kernel bug. And on a second thought, since Clear Linux is a rolling release, so this kind of problem does not persist for long, and won't worth documenting.

> I kind of wanted to avoid compiling a dataset of nvidia-system bugs in the documentation for some reasons

I agree that the documentation is not a dataset for nvidia bugs, which is their own responsibility. But specifically think we need to make it the following known issues:

- *Sometimes*, flatpak applications won't start after updating the driver unless you run `flatpak update`. We can put a `Warning` section right below the 7-th bullet point of `Updating` section.
- Previously, applications like `gnome-control-center` won't start, which is still an open issue here https://github.com/clearlinux/distribution/issues/791. I cannot reproduce this problem when last time I installed the driver, but I'm not sure this is resolved. So it worth mentioning this problem, maybe in the `Troubleshooting` section.
- Most importantly is the problem with hybrid GPUs. And this is the main motivation for opening this issue. But I doubt many people will ignore this part. And comparing to the last two issues, just saying the documentation won't apply to system with hybrid GPUs is not detailed enough. What about showing this?
![](https://aws1.discourse-cdn.com/clearlinux/original/1X/e413fa015036c601ed4f717a4deed01a0c2ebf66.png)
This clearly describes this issue and people won't miss it.
